src/js/Aladin.js

```diff
@@ -1485,15 +1485,25 @@ export let Aladin = (function () {
     Aladin.prototype.removeLayers = Aladin.prototype.removeOverlays;
 
     /**
-     * Remove a overlay by its layer name
-     * 
+     * Remove an overlay by its layer name
+     *
      * @memberof Aladin
-     * @param {string} layer 
+     * @param {string} layer
      */
     Aladin.prototype.removeOverlay = function (layer) {
```

Comment:
I would prefer to have only 1 function removeOverlay that can take either the object (like it does for the moment) or a name. You can have the removeOverlayByName instead in View and not accessible from the user API.
What do you think ?